DIPs/DIP1012.md

```diff
@@ -96,86 +116,71 @@ Encompassed:
 
 Optionally encompassed:
 
-* final/virtual
-* Type_Info / Module Info generation (other components of -betterC?)
+* `final`/virtual
+* `export` -> `dynamicImport`/`dynamicExport`/`neither` 
```

Comment:
That's not what is planned for export, better leave it away.

Reply:
@thewilsonator The talk and DIP were all about **not** introducing an import attribute, but to use export for both. Having an import/export distinction intrinsically relies on compilation boundaries and support from the build systems, so we're looking at simplifying this.
https://github.com/dlang/DIPs/blob/1aaf1d150a58610a7379f016b3ee9e9185e302d3/DIPs/archive/DIP45.md#user-content-a-single-meaning-of-export
http://dconf.org/2016/talks/thaut.pdf